sim/pkg/model/autoscaler.go

```diff
@@ -92,24 +93,38 @@ func NewAutoscaler(env simulator.Environment, startAt time.Time, cluster Cluster
 	return as
 }
 
-const hpaYaml = `
-apiVersion: autoscaling/v2beta2
-kind: HorizontalPodAutoscaler
+//const hpaYaml = `
```

Comment:
Why are you hardcoding a replacement of HPA with VPA?  We need to run one, the other, or both.  Which implies probably a small UI change to select.

One thing you might consider is to put the YAML in the UI as a string, and plumb it through the request.  So the user can edit the yaml as they desire and try things like HPA scale controls.  VPA would just be a second yaml file and maybe a check box.

Something like this: https://github.com/josephburnett/skenario/blob/for-hpa/pkg/serve/index.html#L75-L82